### DISCUSSION TITLE
TypeError: element.waitForDisplayed is not a function

### DISCUSSION THREAD

**Discussion by Urvashi2019** [webdriverio/webdriverio#9916] [General]
getting error when migrate test script from sync to async. Please assist how I can resolve this issue. Thanks 

TypeError: element.waitForDisplayed is not a function
TypeError: element.isrDisplayed is not a function

**Comment by christian-bromann** [comment]
Please make sure `element` is actually a `WebdriverIO.Element` and not something else.
